examples/custom_orientable/custom_orientable/imp.rs

```diff
@@ -62,7 +62,7 @@ impl ObjectImpl for CustomOrientable {
     }
 
     fn dispose(&self) {
-        // Child widgets need to be manually unparented in `dispose()`.
+        // Child widgets need to be manually parentless in `dispose()`.
```

Comment:
this should be un-parented instead.

Reply:
the tool was asking explicitely for parentless :D which seemed weird but I did what the tool wanted blindly. I'll fix it